src/main/webapp/app/overview/course-statistics/course-statistics.component.html

```diff
@@ -46,77 +46,101 @@
+                        </span>
+                        <fa-icon
+                            id="reachable-help"
+                            class="me-1"
+                            [icon]="faQuestionCircle"
+                            ngbTooltip="{{ 'artemisApp.courseOverview.statistics.reachableHelp' | artemisTranslate }}"
+                        ></fa-icon>
+                    </h5>
+                    <h5 class="mb-4 statistics-legend" id="max-course-score">
+                        <span>
+                            {{ 'artemisApp.courseOverview.statistics.totalPoints' | artemisTranslate: { number: overallMaxPoints, relative: totalRelativeScore } }}
+                        </span>
+                        <fa-icon
+                            id="total-help"
+                            class="me-1"
+                            [icon]="faQuestionCircle"
+                            ngbTooltip="{{ 'artemisApp.courseOverview.statistics.totalHelp' | artemisTranslate }}"
+                        ></fa-icon>
+                    </h5>
+                    <div *ngIf="gradingScaleExists" class="mb-5">
+                        <h5 *ngIf="!isBonus" class="statistics-legend">{{ 'artemisApp.courseOverview.statistics.grade' | artemisTranslate }}: {{ gradeDTO?.gradeName }}</h5>
+                        <h5 *ngIf="isBonus" class="statistics-legend">{{ 'artemisApp.courseOverview.statistics.bonus' | artemisTranslate }}: {{ gradeDTO?.gradeName }}</h5>
+                        <h6>
+                            {{ 'artemisApp.courseOverview.statistics.gradeInfo' | artemisTranslate }}
+                            <fa-icon
+                                id="reachable-help"
+                                class="me-1"
+                                [icon]="faQuestionCircle"
+                                ngbTooltip="{{
+                                    isBonus
+                                        ? ('artemisApp.courseOverview.statistics.bonusHelp' | artemisTranslate)
+                                        : ('artemisApp.courseOverview.statistics.gradeHelp' | artemisTranslate)
+                                }}"
+                            ></fa-icon>
+                        </h6>
+                        <div class="mt-3">
+                            <a class="btn btn-primary" [routerLink]="['grading-key']" [queryParams]="{ grade: gradeDTO?.gradeName }">
+                                <fa-icon [icon]="faClipboard"></fa-icon>
+                                <span
+                                    >&nbsp;{{
+                                        isBonus
+                                            ? ('artemisApp.exam.examSummary.gradeKeyButtonBonus' | artemisTranslate)
+                                            : ('artemisApp.exam.examSummary.gradeKeyButton' | artemisTranslate)
+                                    }}</span
+                                >
```

Comment:
```suggestion
                                <span>
                                    {{
                                        isBonus
                                            ? ('artemisApp.exam.examSummary.gradeKeyButtonBonus' | artemisTranslate)
                                            : ('artemisApp.exam.examSummary.gradeKeyButton' | artemisTranslate)
                                    }}
                                </span>
```
Looks like the usual autoformatter problem. I think in most places there is no special non-breaking space between icon and the text. I’m not sure if it is specifically needed here.

Just applying the suggestion will probably make the formatter complain again, it’s probably best to make the change manually.

Reply:
It's always a pleasure for me to get my code reviewed by you but one day we have to evaluate the reasonableness of fighting the autoformatter. 😄 I tried my best to fix the structure manually as you proposed.